config.default

```diff
@@ -122,6 +122,14 @@ overwriteMpasClimatology = False
 #   'bilinear', 'neareststod' (nearest neighbor) or 'conserve'
 mpasInterpolationMethod = bilinear
 
+# should remapping be performed with ncremap or with the Remapper class
+# directly in MPAS-Analysis
+useNcremap = False
+
+# The minimum weight of a denstination cell after remapping, below
+# which it is masked out.
+renormalizationThreshold = 0.1 
+
```

Comment:
With masking, it makes sense to do renormalization so you get an interpolated value with bilinear interpolation even when there are only 1, 2 or 3 bilinear neighbors (though results will often be noisier in these locations, like with nearest neighbor interpolation).  But dividing by the sum of the weights to renormalize only works if that sum isn't too tiny.  This coefficient sets a threshold for how tiny the sum of the weights can be before we consider a cell to be masked out.